src/go/k8s/tests/e2e/produce-consume/00-assert.yaml

```diff
@@ -40,3 +40,28 @@ spec:
             - --default-log-level=debug
 status:
   readyReplicas: 1
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cluster-sample
+spec:
+  clusterIP: None
+  ports:
+    - name: kafka-tcp
+      port: 9092
+      protocol: TCP
+      targetPort: 9092
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cluster-sample-external
+spec:
+  ports:
+    - name: kafka-tcp
+      port: 9092
```

Comment:
so we're not using the kafkaPort+1 anymore?

Reply:
This is prelude to the actual implementation. The following pieces need ot be merged:
- init container 
- node port 
- host path 
- advertised kafka api internal and external

Next PR will address this.

Reply:
okay I don't fully follow it then, are we merging something that does not work? or perhaps I don't see the whole flow. Okay, I'll trust you here...

Reply:
We agree on splitting the PRs to not introduce one big thing. You have comments in the code https://github.com/vectorizedio/redpanda/pull/756/files#diff-d8d3d68da6374674c5a5fae39e3220fdfa6a581caa5ef01f675aa3757f9a9d80R92-R116 that stays the service type node port does nothing. It only help later when we connect this port to host port in the container port specification (NEXT PRs).

Should I do something more? Rephrase something?